docs/libertyRun.md

```diff
@@ -0,0 +1,38 @@
+## libertyRun task  
+Start a Liberty server in foreground. The server instance will be automatically created if it does not exist.  
```

Comment:
Hmmm, since we are not using the ant task for run, this second sentence is not true, but we should add it.  If you look at the ant ServerTask.java, you see that the run task starts off by checking:
```
        if (!serverConfigDir.exists()) {
            log(MessageFormat.format(messages.getString("info.server.create"), serverName));
            doCreate();
        }
```
We need to do something similar and call the create task, if the server configuration does not exist.  It should in turn call install, if the runtime does not exist.